src/app/components/Transcript/index.styles.tsx

```diff
@@ -0,0 +1,64 @@
+import { css, Theme } from '@emotion/react';
+
+export default {
+  transcript: ({ spacings, palette, isDarkUi }: Theme) =>
+    css({
+      backgroundColor: isDarkUi ? palette.GREY_7 : palette.WHITE,
+      display: 'block',
+      marginBottom: `${spacings.TRIPLE}rem`,
+      padding: `${spacings.DOUBLE}rem`,
+      color: palette.GREY_6,
+    }),
+
+  summary: ({ palette, isDarkUi }: Theme) =>
+    css({
```

Comment:
Missing cursor: pointer. 
Also can we move the padding from the details element to here so the touch target is larger please